tests/test_api/test_item.py

```diff
@@ -19,6 +26,38 @@ def test_get_item_missing(self, test_client, item):
         check_valid_header_type(r.headers)
         assert r.status_code == 404
 
+    def test_put_item_upvote(self, test_client, user, item):
+        self.login(test_client, user.email, user.password)
+
+        upvote = dict(vote=1)
```

Comment:
If we make vote a Bool, rather than an int (true=upvote, false=downvote), we won't have to worry about someone making a call to the api with >1 vote.

Or, maybe we can look into how to set a min and max on the vote?

Not sure what makes the most sense.